.github/workflows/releases-json.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04
         with:
           base: master
-          branch: releases-json/${{ github.event.release.name }}
+          branch: bot/releases-json
```

Comment:
Would prefer if we could do something like a date? Just to disambiguate between multiple.